v1/pkg/client/api/openapi.yaml

```diff
@@ -3112,16 +3112,16 @@ components:
           type: array
       type: object
     AllocationStorage:
-      description: allocation of storage for each PCE service.
+      description: Allocation of storage for each PCE service.
       example:
         InstanceFamily: Performance Optimized Block
         AllocationInfo:
         - AllocatedCount: 1
           ServiceType: BMaaS
-          AllocatedCapacity: 5
+          AllocatedCapacity: 0.5962134
```

Comment:
Are these values valid w.r.t the 'Total' value?

Reply:
Yes. Total capacity is total storage capacity. Allocated is sum of all volumes allocated essentially